### PR TITLE
Setting Up A/B Test for G Suite Upsell

### PR DESCRIPTION
**Pull request by taggon**
This PR will set up an A/B test for G Suite upsell to get more users use G Suite upgrade.
See p7EOS0-3aB-p2 for more detail.

## Test plan

* Set the _gsuiteUpsell_ A/B test to _show_.
* Create a new site with a custom domain, not under wordpress.com.
* You should be able to see the G Suite upsell page after checking out the cart as follows:
screen shot 2017-10-25 at 23 10 19
* The thank you page shows up when you click on the cancel link.
* If you want to upgrade your site with G Suite, click on the blue button at the footer, fill out the form and push the "Continue" button, then you will see the check out page again. If everything goes well, you will be at the thank you page with G Suite account setup message.
screen shot 2017-10-25 at 23 36 52

cc @fditrapani @anneforbush

### CODE REVIEW

**Comment by fditrapani** [comment]
Looks good @taggon. Here are some notes going through the flow:

- We'll need to make some text/copy changes based on the new design. The changes can be applied both to the test and control. This could also be done in a separate PR.
- These are the flows I tested with some notes.
  - When I bought a domain and a plan, it showed the up sell. ✅ 
    - After I add my emails I see a [new screen](https://cloudup.com/cVwafVP_mDj) I haven't seen before. Is this screen necessary? It doesn't show if I try the regular domain purchase flow within Calypso. Can it be removed?  ❌
    - I'm asked to pick another plan when I add G Suite to a personal plan with a custom domain. It doesn't seem to like the personal plan.  ❌
    - When I get to the checkout after adding G Suite, I see the [Premium Plan](https://cloudup.com/c2n7NUfp67Y) in my cart again. It actually double charges me for the plan — how weird. If I try checkout with the Premium plan it shows me this message: `You are attempting to purchase new products with renewals. Please purchase them separately.` At this point, G Suite package should be the only thing in the cart. ❌
  - When I bought a plan without a domain, it didn't show. ✅
  - When I bought a domain in calypso after signup, the regular flow remained in tact. ✅
  -  When I bought a domain through WordPress.com/domains, the up sell didn't show. Technically, it should be showing here as well but I think we're ok without it for the test because /domains isn't fully public yet (There is no link to it in the main menu on WordPress.com) ✅
  - Clicking on "No thanks" took me to the thank you page. ✅

**Comment by fditrapani** [comment]
@taggon I retested the flows again and it's looking great. I updated my previous comment to reflect the updates. The only two things remaining here are:

- New hangout logo, it looks like you added the old one again. You can [download the new logo here](https://cdn.zapier.com/storage/photos/087699d6aac700a10ede8ad57607f0d4.png).
- I still see [this screen](https://cloudup.com/cVwafVP_mDj) after I enter my email addresses into G Suite up sell screen. This screen does not show if I purchase a domain in Calypso and opt in for G Suite — [view screen grab](https://cloudup.com/cMdzDOEJrZn)

**Comment by fditrapani** [comment]
Thanks for the updates @taggon. I tested it out and it works! Thank you.

File: client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ class GoogleAppsProductDetails extends Component {
 
 					<p>
 						{ translate(
-							"We've partnered with Google to offer you email, storage, docs, calendars " +
+							"We've teamed up with Google to offer you email, storage, docs, calendars, " +
```
Comment (a8ci18n) [inline client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'We've partnered with Google to offer you email, storage, docs, calendars, and more integrated with your site.' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/abtest-gsuite-in-signup-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by fditrapani** [comment]
Looks good now 👍

**Pull request merged**